src/main.py

```
@@ -1,253 +1,206 @@
-            return html_response(docs_decrypt())
-        
-        # Admin page
-        if request.method == "GET" and url.pathname == "/admin/onboard":
-            return html_response(
-                admin_onboard_page(env.TURNSTILE_SITE_KEY if ts_enabled else "")
-            )
-        
-        # Admin onboard POST
-        if request.method == "POST" and url.pathname == "/admin/onboard":
+                return html_response(html)
+            except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)